wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js

```diff
@@ -8,6 +8,18 @@ const WorkspaceSwitcherPopupList = Self.imports.workspacePopup.workspaceSwitcher
 
 var modals = [];
 
+function primaryModifier(mask) {
```

Comment:
This can't just be imported because it's not assigned a `var`, right? Have you tried?